multitenantcy/tenant-manager-api/src/main/java/io/apicurio/multitenant/persistence/RegistryTenantRepositoryImpl.java

```diff
@@ -0,0 +1,42 @@
+package io.apicurio.multitenant.persistence;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import io.apicurio.multitenant.persistence.dto.RegistryTenantDto;
+import io.apicurio.multitenant.persistence.hibernate.RegistryTenantPanacheRepository;
+
+@ApplicationScoped
+public class RegistryTenantRepositoryImpl implements RegistryTenantRepository {
+
```

Comment:
What's the main purpose of this wrapper? To hide the other available methods in the Panache repository?

Reply:
to hide the rest of the methods from Panache repository and to help as an abstraction from the storage libraries used